datafusion/core/src/physical_plan/sorts/sort.rs

```diff
@@ -1089,7 +820,7 @@ mod tests {
     #[tokio::test]
     async fn test_sort_fetch_memory_calculation() -> Result<()> {
         // This test mirrors down the size from the example above.
-        let avg_batch_size = 5000;
+        let avg_batch_size = 4000;
```

Comment:
I had to adjust this down slightly, as the memory characteristics have changed slightly (for the better)